express_cut/express_service/views.py

```diff
@@ -91,5 +91,58 @@ def users_views(request, pk):
     return Response(status=status.HTTP_500_INTERNAL_SERVER_ERROR)
 
 
+@swagger_auto_schema(methods=['POST'], request_body=DailyScheduleSerializer, responses=swagResp.commonPOSTResponses,
+                     tags=['dailySchedule'], )
+@api_view(['POST'])
+@authentication_classes([SessionAuthentication, BasicAuthentication])
+@permission_classes([IsAuthenticated])
+def schedule_views(request):
+    if request.method == 'POST':
+        data = request.data
+        serializer = DailyScheduleSerializer(data=data)
+        # Checks if the user logged in is a manager & if it's logged in.
+        if not DailySchedulePermissions().POST_permissions(request):
+            return Response(status=status.HTTP_403_FORBIDDEN)
+        # Checks if you have both a date and a stylist id.
+        if not serializer.is_valid():
+            return Response({"message": serializer.errors}, status=status.HTTP_400_BAD_REQUEST)
+        try:
+            schedule = serializer.save()
+            return Response(data={'pk': schedule.pk}, status=status.HTTP_201_CREATED)
+        except:
+            return Response(status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@swagger_auto_schema(methods=['GET', 'DELETE'], responses={**swagResp.commonResponses, **swagResp.getResponse(UserSerializer)},tags=['dailySchedule'],)
+@swagger_auto_schema(methods=['PUT'], request_body=DailyScheduleSerializer, responses=swagResp.commonPOSTResponses,
+                     tags=['dailySchedule'], )
+@api_view(['GET','PUT', 'DELETE'])
+@authentication_classes([SessionAuthentication, BasicAuthentication])
+@permission_classes([IsAuthenticated])
+def schedule_views_put(request, pk):
+    try:
+        schedule = DailySchedule.objects.get(pk=pk)
```

Comment:
I don't see that your are managing permissions here. Everyone shouldn't have permissions for this. They should look something like this:

GET - Managers, Customers and the stylist that is related to that schedule.
PUT- Managers
DELETE - Managers

Reply:
Everyone should be able to see every Stylist schedule. Will handle permissions for PUT and DELETE.